src/CreateNewPage/index.js

```diff
@@ -140,57 +166,29 @@ class CreateItemPage extends Component {
             onSubmit={this.createItem}
             aria-labelledby="create-item"
           >
-            <TextField
-              ref={id => (this.itemId = id)}
-              id="itemId"
-              label={`${unit} ID`}
-              required
-              type="text"
-              error={idError}
-              errorText="This field is required."
-            />
-            <SelectField
-              ref={departure => (this.departure = departure)}
-              id="departure"
-              required
-              label="Departure Port"
-              className="md-cell"
-              menuItems={PORTS}
-              position={SelectField.Positions.BELOW}
-              error={departureError}
-              errorText="This field is required."
-              dropdownIcon={<FontIcon>expand_more</FontIcon>}
-            />
-            <SelectField
```

Comment:
why these fields were removed?

Reply:
we agreed to only ignore the specific configuration of the master branch, that is why this change coming from window